src/pages-and-resources/discussions/DiscussionsSettings.jsx

```diff
@@ -16,18 +17,24 @@ import { Check } from '@edx/paragon/icons';
 import FullScreenModal from '../../generic/full-screen-modal';
 import Stepper from '../../generic/stepper';
 import { PagesAndResourcesContext } from '../PagesAndResourcesProvider';
-import AppList from './app-list';
-import AppConfigForm from './app-config-form';
+
 import messages from './messages';
 import DiscussionsProvider from './DiscussionsProvider';
+import { fetchApps } from './data/thunks';
+import AppList from './app-list';
+import AppConfigForm from './app-config-form';
 
 function DiscussionsSettings({ courseId, intl }) {
+  const dispatch = useDispatch();
   const { path } = useRouteMatch();
   const { pathname } = useLocation();
-
   const { path: pagesAndResourcesPath } = useContext(PagesAndResourcesContext);
-  const discussionsPath = `${pagesAndResourcesPath}/discussions`;
 
+  useEffect(() => {
+    dispatch(fetchApps(courseId));
```

Comment:
We pull fetchApps back up into DiscussionsSettings and then use the data loaded by it for both the AppList and AppConfigForm.  That means the requests in those two components go away.